drawing_1.css

```diff
@@ -0,0 +1,26 @@
+.container{
+	width: 250px;
+	height: 100px;	
```

Comment:
What happens when you don't set a width and height for `.container`. Play around with it and see if it affects anything.